features/partner/partner_assessment.feature

```diff
@@ -81,7 +81,7 @@ Scenario: An applicant and partner's combined capital is over the lower threshol
     When I retrieve the final assessment
     And I should see the following overall summary:
       | attribute                  | value                 |
-      | assessment_result          | contribution_required |
+      | assessment_result          | ineligible            |
```

Comment:
I'm intrigued: why does this change?

Reply:
So, the test was phrased as a "standard" assessment, but the relevant step definition had it as using a domestic abuse proceeding type, which is really a special case. To accurately describe the bug scenario I fixed the step definition so that for a "standard" assessment we don't use the special case proceeding type. So for the scenario described here, if this really was a standard assessment the result would always have been 'ineligible'. It was only marked as 'contribution_required' due to the effects of the proceeding type special case.

Reply:
Ah super explanation, ta. I had missed that the step definition change of code was for this same test